src/features/encoders/mozJPEG/shared/meta.ts

```diff
@@ -37,3 +37,67 @@ export const defaultOptions: EncodeOptions = {
   separate_chroma_quality: false,
   chroma_quality: 75,
 };
+
+/**
+ * Simple helper function that wraps each line of the input at `line_width` characters (default 60)
+ */
+function wrapText(text: string, line_width: number = 60): string {
+  const lines = text.split('\n');
+  let balancedText = '';
+  for (const line of lines) {
+    balancedText += '\n';
+    const words = line.split(' ');
+    let runningLength = 0;
+    for (const word of words) {
+      if (runningLength + word.length + 1 > line_width) {
+        balancedText += '\n' + word + ' ';
+        runningLength = word.length;
+      } else {
+        runningLength += word.length + 1;
+        balancedText += word + ' ';
+      }
+    }
+  }
+  return balancedText.trim();
+}
```

Comment:
This function is for the UI and should probably in `src/client/lazy-app/util` or something.

Reply:
I also think we can simplify this. The algorithm here should probably be something like

- Split text into arrays of lines
- For each line:
  - While current output line is shorter than `limit`
    - Add the next word from the current line to the current output line 
  - Emit current output line
  - Set output line to empty string.

Reply:
Ahh you're right, it would be better suited for the utilities file. I'll make a generator version as well.